### ISSUE TITLE
No sensors att all

### ISSUE THREAD

**Issue by BoteBFKJ** [nlrb/com.weather-sensors#119]
Hello!
After removing all sensors according to the instructions below, I can't get a single sensor in anymore. I have uninstalled app one etc. Before I had around 15 that could be added. What to do?

Sincerely
Frederick

**Comment by nlrb** [comment]
What do you see in the app settings? Can you add a screenshot?

**Comment by BoteBFKJ** [comment]
![IMG_3908](https://github.com/nlrb/com.weather-sensors/assets/121239017/c97e42a3-82ef-4925-b8a9-41421419150c)

**Comment by nlrb** [comment]
Are all 15 sensors only with these 2 protocols? Maybe PTP on Homey? No idea what was received before and why it would not work anymore.

**Comment by BoteBFKJ** [comment]
![image](https://github.com/nlrb/com.weather-sensors/assets/121239017/f0fc809b-ef32-42f7-88da-3aa8c1e244f9)

**Comment by BoteBFKJ** [comment]
I have tried this with, unfortunately it makes no difference.
Is there any cache to clear?

//Fredrik

**Comment by nlrb** [comment]
No, there is no cache or anything to clear. The Lacrosse protocol has also not changed across versions. Have you tried with fresh batteries in a sensor?

**Comment by BoteBFKJ** [comment]
Hello
Sorry for the late reply but it seems they appeared as suddenly as they disappeared. It took a couple of days before they were back.
Thanks for a great app.
//Fredrik
